packages/babel-plugin/src/__tests__/expression-evaluation.test.tsx

```diff
@@ -276,4 +276,19 @@ describe('import specifiers', () => {
       '{background-color:blue}',
     ]);
   });
+
+  it('should not blow up when member expression object is other than "Identifier" or "Call Expression"', () => {
+    expect(() => {
+      transform(`
+      import '@compiled/core';
+      import React from 'react';
+
+      function Component() {
+        return (
+          <span css={{ width: [{ bar: 10 }][0].bar }} />
+        );
+      };
+    `);
+    }).not.toThrow();
```

Comment:
what comes out instead? a CSS variable/dynamic value?

Reply:
Let me check

Reply:
It should be a variable.

Reply:
```ts
import{ax,CC,CS}from'@compiled/core';import React from'react';const _="._1bsb11sr{width:var(--_1mrga3f)}";function Component(){return<CC>
  <CS>{[_]}</CS>
  {<span className={ax(["_1bsb11sr"])}style={{"--_1mrga3f":[{bar:10}][0].bar}}/>}
</CC>;};
```

Reply:
I mean same variable as input. No conversion.